auth0/src/main/java/com/auth0/android/authentication/AuthenticationAPIClient.java

```diff
@@ -191,6 +212,12 @@ public void setUserAgent(String userAgent) {
      */
     @SuppressWarnings("WeakerAccess")
     public AuthenticationRequest login(@NonNull String usernameOrEmail, @NonNull String password, @NonNull String connection) {
```

Comment:
What I'd do is to chose the grant type and legacy endpoint based on the pipeline flag. If it's off, ro and connection is used otherwise token and realm